test/sagemaker_tests/pytorch/training/integration/sagemaker/test_mnist_inductor.py

```diff
@@ -29,7 +29,7 @@
 @pytest.mark.integration("smexperiments")
 @pytest.mark.skip_gpu
 @pytest.mark.skip_inductor_test
-@pytest.mark.xfail(reason="known issue: https://github.com/pytorch/pytorch/issues/98436")
+@pytest.mark.skip(reason="known issue: https://github.com/pytorch/pytorch/issues/98436")
```

Comment:
Do we want to change from `xfail` to `skip`? In the future, in case the issue is resolved and the tests start to pass, if we use `skip`, we would not be able to catch the passing tests and would not be able to rerun the tests. 

In other words, with `skip` we might never get to run the tests even when the issue is resolved.

Reply:
If the issue is resolved in the future, we'll need to unskip these tests.

As of now, due to the use of `xfail`, the overall behavior of the test job is exactly the same regardless of whether these inductor tests pass or fail, but we waste valuable time and money on running a test that we know is going to fail.

I've created a GitHub Issue to track this pending task - https://github.com/aws/deep-learning-containers/issues/3128

Right now, until we remove the xfail marker, we'll never know if the test is working correctly anyway.